helper-cli/src/main/kotlin/commands/ImportLicenseClassificationsCommand.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands
+
+import com.fasterxml.jackson.module.kotlin.readValue
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.options.split
+import com.github.ajalt.clikt.parameters.types.enum
+import com.github.ajalt.clikt.parameters.types.file
+
+import java.net.URL
+
+import org.apache.logging.log4j.kotlin.Logging
+
+import org.ossreviewtoolkit.model.jsonMapper
+import org.ossreviewtoolkit.model.licenses.LicenseCategorization
+import org.ossreviewtoolkit.model.licenses.LicenseCategory
+import org.ossreviewtoolkit.model.licenses.LicenseClassifications
+import org.ossreviewtoolkit.model.yamlMapper
+import org.ossreviewtoolkit.utils.common.expandTilde
+import org.ossreviewtoolkit.utils.ort.ORT_LICENSE_CLASSIFICATIONS_FILENAME
+import org.ossreviewtoolkit.utils.ort.ortConfigDirectory
+import org.ossreviewtoolkit.utils.spdx.SpdxSingleLicenseExpression
+
+internal class ImportLicenseClassificationsCommand : CliktCommand(
```

Comment:
The command is doing multiple things at once.

1. Importing / mapping the models of the providers to the Ort model
2. Renaming categories / adding prefixes
3. Merging license classification files

To allow for more use cases, I propose to change the import command to import classifications from a single provider to a new empty `license-classifications.yml` file. E.g. `orth import-license-classifications --provider ECLIPSE`.

Then also have a dedicated merge command, which allows merging multiple license classification files.

Reply:
I would have preferred to avoid the overhead of creating multiple commands. But I can see the benefit of having a separate merge command. I'll give it a try to implement this via more options to the same command.

Reply:
Maybe even 3: import, merge, rename. 
This will keep things simple and allow for more flexibility. Integrating a flexible renaming into the merge command IMO is way more complicated that having a separate rename categories command.

Reply:
> This will keep things simple

Actually, this will add more complexity to the users at least I have in mind, as they now need to call 2 / 3 tools with arguments instead of just one.

> allow for more flexibility.

For a balance between flexibility and ease of use, I'm now planning for 2 commands: import and merge. The import will take an optional `--prefix` for the renaming / scoping of categories.